frontend/src/AddSeries/ImportSeries/SelectFolder/ImportSeriesSelectFolder.js

```diff
@@ -78,7 +78,10 @@ class ImportSeriesSelectFolder extends Component {
                       Make sure that your files include the quality in their filenames. eg. <span className={styles.code}>episode.s02e15.bluray.mkv</span>
                     </li>
                     <li className={styles.tip}>
-                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span>
+                      Point Sonarr to the folder containing all of your tv shows, not a specific one. eg. <span className={styles.code}>"{isWindows ? 'C:\\tv shows' : '/tv shows'}"</span> and not <span className={styles.code}>"{isWindows ? 'C:\\tv shows\\the simpsons' : '/tv shows/the simpsons'}"</span> Additionally, each movie must be in its own folder within the root/library folder.
```

Comment:
Is movie support coming in a separate PR?

Reply:
pretty sure there's no plans for that feature at this time xD